CHANGELOG.md

```diff
@@ -4,8 +4,14 @@
 
 ## [Unreleased]
 
-### Added
-### removed unwanted/extra lines from Readme.md file
+### Removed
```

Comment:
For some reason, I cannot propose changes in file,  please update this file like this


```md
<!-- Keep a Changelog guide -> https://keepachangelog.com -->

# GitHub-Theme Changelog

## v0.0.2 [ 14-aug-2021 ]

### Removed [ 14-aug-2-2021 ]
- removed unwanted/extra lines from Readme.md file

### Changed [ 14-aug-2-2021 ]

- GitHub Actions: Use Java 11
- Update `pluginVerifierIdeVersions` to `2020.3.4, 2021.1.3`
- Change since/until build to `203-212.*`
- Upgrade Gradle Wrapper to `7.1.1`

## v0.0.1 [ 10-aug-2021 ]

### Added

- chore: Setup Project
- Initial scaffold created from [IntelliJ Platform Plugin Template](https://github.com/JetBrains/intellij-platform-plugin-template)

```